05_stringer/satish/main_test.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestMainOutput(t *testing.T) {
+	// Given
+	r := require.New(t)
+	var buf bytes.Buffer
+	out = &buf
+
+	// When
+	main()
+
+	// Then
+	expected := "127.0.0.1\n"
+	actual := buf.String()
+	r.Equalf(expected, actual, "Unexpected output in main()")
+}
+
+// var tests = []struct {
```

Comment:
Remove commented text.
Add `TestIPAddrString` and use [table driven tests](https://github.com/golang/go/wiki/TableDrivenTests) for different string output comparisons.